### ISSUE TITLE
[BUG] Some cursors exist yet aren't applied apparently

### ISSUE THREAD

**Issue by umarhamzy** [ful1e5/bibata#36]
Upon testing the cursors on https://vibhorjaiswal.github.io/Cursor-Test/, some items are missing such as
- alias
- cell
- col-resize
- context-menu
- copy
- grab
- grabbing
- row-resize
- zoom-in
- zoom-out

I understand that most of these cursors are probably not possible to implement unless you manually do them, but things like zoom-in, zoom-out, grab, grabbing already exist in the file that we download, yet isn't applied it seems.

**Comment by ful1e5** [comment]
@umarhamzy Is it still the case with the latest version (currently v2.0.7)?

**Comment by umarhamzy** [comment]
I downloaded the package from the web, but from the naming it seems its still the old version? (Bibata-Modern-11727-v1.0.2-win). Should I download from GitHub? Please check and let me know so I can test this. I'm currently on v1.0.1 from what I can tell, and the case remains the same

Also, I wanted to use the web so I can download in 22px which fits the best for me

**Comment by stanio** [comment]
>   zoom-in, zoom-out, grab, grabbing already exist in the file

Windows doesn't have such standard cursors.  See your "Mouse Properties" control panel – the "Pointers" tab.  Different applications may provide similar cursors on their own but there's no standard way of customizing them.

**Comment by umarhamzy** [comment]
> > zoom-in, zoom-out, grab, grabbing already exist in the file
> 
> Windows doesn't have such standard cursors. See your "Mouse Properties" control panel – the "Pointers" tab. Different applications may provide similar cursors on their own but there's no standard way of customizing them.

Yup, I've come to this conclusion some time ago after having checked mouse properties. I still don't understand how this could be resolved
